### PR TITLE
Docker way

### PR DESCRIPTION
**Pull request by Storager**
A small adjustment in order to help people who do not want to have PHP installed locally.

### CODE REVIEW

**Comment by Storager** [comment]
docker hub build: https://hub.docker.com/r/storager/shapeup

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (j6s) [inline entrypoint.sh]: Would you mind renaming `entrypoint.sh` to `docker-entrypoint.sh` to communicate more clearly what the intention of the file is

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+echo "========== Downloading single HTML =================="
+php /app/cli.php download:single-html
+echo "============ Converting to e-book ===================="
+ebook-convert shape-up.html /app/output/shape-up.${1} \
```
Comment (j6s) [inline entrypoint.sh]: While I see the intention behind this, I would rather not bundle this functionality with the library: When it comes to books, rights are a bit complicated. I can see a future where 37signals decides to sell an epub version of the file and I don't want it to seem like I am trying to do the same thing for free.

The scope of this library should only be "if you want this offline, here you go".

While converting it automatically probably is fine, I just don't want that risk in this repository. I hope you understand.

File: README.md [hunk 3]
```diff
@@ -40,3 +40,14 @@ The code to download it yourself is here, but you must use it yourself.
...
 That's it.
+
+## Docker way
+
+There is a way to not deal with php at all. For that it is enough to have docker engine installed.
+
+Usage
+```
+docker build -t shapeup .
+docker run -it --rm -v $(pwd)/output:/app/output shapeup epub
```
Comment (j6s) [inline README.md]: Can you change this from building the image to just running `thej6s/shape-up-downloader` ?
I'll setup docker hub to autobuild as soon as this PR is merged.

**Review by j6s — CHANGES_REQUESTED** [review]
Nice, thank you.

There are a couple of small change requests I have, but all in all this seems good.

File: entrypoint.sh (hunk 1 above)
Comment (Storager) [reply entrypoint.sh]: entrypoint.sh is a common naming for docker development, but I can rename it.

**Pull request merged**